### ISSUE TITLE
Capture geographic coordinates based on the pointers

### ISSUE THREAD

**Issue by patricechataigner** [oiime/sidr#17]
Is it possible to create two field for longitude and latitude and to capture those points based on the pointer? same for the polygons or the radius.

**Comment by oiime** [comment]
You mean manually enter them and show them on the map? I think you can paste it in the search bar on the top

**Comment by patricechataigner** [comment]
Not exactly. when we geolocate the information in the google map, is it possible that the system record the latitude and longitude of the coordinates and stor them so they appear in the csv export (one column lat and one column long)

**Comment by patricechataigner** [comment]
Not exactly. when we geolocate the information in the google map, is it possible that the system record the latitude and longitude of the coordinates and stor them so they appear in the csv export (one column lat and one column long)
